demos/classification_demo/python/classification_demo.py

```diff
@@ -0,0 +1,308 @@
+    frame = output_transform.resize(frame)
+    class_id = classifications[0][0]
+    label_height = cv2.getTextSize(labels[class_id], cv2.FONT_HERSHEY_COMPLEX, 0.75, 2)[0][1]
+    initial_labels_pos =  frame.shape[0] - label_height * (2 * len(classifications) + 1)
+
+    if (initial_labels_pos < 0):
+        initial_labels_pos = label_height
+        log.warning('Too much labels to display on this frame, some will be omitted')
+    offset_y = initial_labels_pos
+
+    for cl in classifications:
+        class_id = cl[0]
+        conf = cl[1]
+        label = '{} {:.1%}'.format(labels[class_id], conf)
+        label_width = cv2.getTextSize(label, cv2.FONT_HERSHEY_COMPLEX, 0.75, 2)[0][0]
+        offset_y += label_height * 2
+        color = (10, 210, 10) if gt_idx == class_id else (10, 10, 210)
+        put_highlighted_text(frame, label, (frame.shape[1] - label_width, offset_y),
+            cv2.FONT_HERSHEY_COMPLEX, 0.75, color, 2)
+    return frame
+
+
+def print_raw_results(classifications, frame_id, labels, gt_id):
+    label_max_len = len(max([labels[cl[0]] for cl in classifications], key=len))
+    log.debug(' ------------------- Frame # {} ------------------ '.format(frame_id))
+    if gt_id:
+        log.debug(' Ground Truth: {} {}'.format(gt_id, labels[gt_id]))
+
+    log.debug(' Class ID | {:^{width}s}| Confidence '.format('Label', width=label_max_len))
+    for cl in classifications:
+        class_id = cl[0]
+        conf = cl[1]
+        log.debug('{:^9} | {:^{width}s}| {:^10f} '.format(class_id, labels[class_id], conf, width=label_max_len))
+
+
+def load_ground_truth(gt_file, image_names, classes_num):
+    with open(gt_file, 'r') as f:
+        ground_truth = {}
+        for s in f:
+            separator_idx = s.find(' ')
+            if (separator_idx == -1):
+                raise Exception('The labels file has incorrect format.')
+            ground_truth[s[0:separator_idx]] = s[separator_idx + 1:]
+
+    indices = []
+    for name in image_names:
+        try:
+            idx = int(ground_truth[name])
+            if idx > classes_num:
+                raise Exception('Class index {} is outside the range supported by the model.'.format(idx))
+            indices.append(int(ground_truth[name]))
+        except KeyError:
+            raise Exception('No class specified for image ' + name)
+
+    return indices
+
+
+def main():
+    args = build_argparser().parse_args()
+
+    cap = open_images_capture(args.input, args.loop)
+
+    log.info('OpenVINO Inference Engine')
+    log.info('\tbuild: {}'.format(get_version()))
+    ie = IECore()
+
+    plugin_config = get_user_config(args.device, args.num_streams, args.num_threads)
+
+    log.info('Reading model {}'.format(args.model))
+    model = models.Classification(ie, args.model, ntop=args.ntop, labels=args.labels, logger=log)
+    log_blobs_info(model)
+
+    gt_indices = None
+    if args.ground_truth and isinstance(cap, DirReader):
```

Comment:
```suggestion
    if args.ground_truth and cap.get_type() == 'DIR':
```
I think the demo should reports the warning message when a user submits a video/image/camera_id as demo input and ground truth indices via `--ground_truth`.  Also, the documentation should say that the key is applied only with the folder input

Reply:
Done in 4ac5f7e.